java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -58,12 +58,27 @@ public static int bitIndex(int absoluteBitIndex) {
   public static void setValidityBitToOne(ArrowBuf validityBuffer, int index) {
     final int byteIndex = byteIndex(index);
     final int bitIndex = bitIndex(index);
-    byte currentByte = validityBuffer.getByte(byteIndex);
-    final byte bitMask = (byte) (1L << bitIndex);
+    int currentByte = validityBuffer.getByte(byteIndex);
```

Comment:
please provide a comment on why you are assigning a byte to an int.

Reply:
Good point. Comment added.